scout/server/blueprints/variants/views.py

```diff
@@ -376,9 +382,12 @@ def cancer_sv_variants(institute_id, case_name):
     page = int(request.form.get("page", 1))
     variant_type = request.args.get("variant_type", "clinical")
     category = "cancer_sv"
-
     # Define case and institute objects
     institute_obj, case_obj = institute_and_case(store, institute_id, case_name)
+    variants_stats = case_obj.get("variants_stats") or store.case_variants_count(
+        case_obj["_id"], institute_id, True
+    )
+
```

Comment:
4

🙂

Reply:
Good idea, I'll fix as you suggested, thanks for the review!

Reply:
Done, I have fixed store.case_variants_count so it does everything in one place. I'm goin to install the branch on stage and give it a try there!